jax/random.py

```diff
@@ -311,41 +313,78 @@ def shuffle(key, x, axis=0):
   return x
 
 
-@partial(jit, static_argnums=(1, 2))
-def normal(key, shape, dtype=onp.float32):
-  """Sample standard normal random values with given shape and float dtype.
+@partial(jit, static_argnums=(3, 4))
+def normal(key, loc=0.0, scale=1.0, shape=(), dtype=onp.float32):
+  """Sample normal random values with given shape, mean, variance and float dtype.
 
   Args:
     key: a PRNGKey used as the random key.
-    shape: a tuple of nonnegative integers representing the shape.
+    loc: optional, the mean of the distribution (default 0.0).
+    scale: optional, the standard deviation of the distribution (default 1.0).
+    shape: optional, a tuple of nonnegative integers representing the shape 
+      (default scalar).
     dtype: optional, a float dtype for the returned values (default float32).
 
   Returns:
     A random array with the specified shape and dtype.
   """
+  if not onp.issubdtype(dtype, onp.floating):
+    raise TypeError("uniform only accepts floating point dtypes.")
+  loc = onp.array(loc, dtype)
+  scale = onp.array(scale, dtype)
+
   lo = onp.nextafter(onp.array(-1., dtype), 0., dtype=dtype)
   hi = onp.array(1., dtype)
-  u = uniform(key, shape, dtype, lo, hi)
-  return onp.array(onp.sqrt(2), dtype) * lax.erf_inv(u)
+  u = uniform(key, lo, hi, shape, dtype)
+  return loc + scale * onp.array(onp.sqrt(2), dtype) * lax.erf_inv(u)
 
 
-@partial(jit, static_argnums=(2,))
-def bernoulli(key, mean=onp.float32(0.5), shape=()):
+@partial(jit, static_argnums=(3, 4))
+def multivariate_normal(key, mean, cov, shape=(), dtype=onp.float32):
+  """Sample normal random values with given shape, mean, variance and float dtype.
+
+  Args:
+    key: a PRNGKey used as the random key.
+    mean: the mean of the distribution.
+    cov: the variance of the distribution.
+    shape: optional, given a shape of e.g. (m, n, k) for an N-d distribution, the
+      output shape will be (m, n, k, N) [default: (), which corresponds to a single
+      N-d sample].
+    dtype: optional, a float dtype for the returned values (default float32).
+
+  Returns:
+    A random array with the specified shape and dtype.
+  """
+  if not onp.issubdtype(dtype, onp.floating):
+    raise TypeError("uniform only accepts floating point dtypes.")
+  mean = onp.array(mean, dtype)
+  cov = onp.array(cov, dtype)
+  L = np.linalg.cholesky(cov)
+  dim = mean.shape[0]
+
+  lo = onp.nextafter(onp.array(-1., dtype), 0., dtype=dtype)
+  hi = onp.array(1., dtype)
+  u = uniform(key, lo, hi, (*shape, dim), dtype)
+  return mean + np.dot(onp.array(onp.sqrt(2), dtype) * lax.erf_inv(u), L.T)
```

Comment:
I guess this is `np.dot(L, vector)` instead of `np.dot(vector, L.T)`? In addition, we can simplify the implementation by taking an `eps = normal(key, shape, dtype)`.